claat/render/html.go

```diff
@@ -132,7 +137,8 @@ func (hw *htmlWriter) writeFmt(f string, a ...interface{}) {
 }
 
 func (hw *htmlWriter) writeEscape(s string) {
-	htmlTemplate.HTMLEscape(hw.w, []byte(s))
+	s = htmlTemplate.HTMLEscapeString(s)
```

Comment:
Do you want = or := ?

Reply:
`=` since `s` is already defined (I was getting a compile error with `:=`)?